pkg/storaged/stratis-details.jsx

```diff
@@ -207,6 +216,153 @@ export const StratisPoolDetails = ({ client, pool }) => {
         });
     }
 
+    function add_passphrase() {
+        dialog_open({
+            Title: _("Add passphrase"),
+            Fields: [
+                PassInput("passphrase", _("Passphrase"),
+                          { validate: val => !val.length && _("Passphrase cannot be empty") }),
+                PassInput("passphrase2", _("Confirm"),
+                          { validate: (val, vals) => vals.passphrase.length && vals.passphrase != val && _("Passphrases do not match") })
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230529-121503-eadabc01-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)